src/commands/add/spec/add-target.spec.ts

```diff
@@ -15,6 +15,15 @@ import { TargetJson } from '../../../types/configuration'
 import { TargetScope } from '../../../types/targetScope'
 import { createTestMinimalApp, removeTestApp } from '../../../utils/test'
 
+interface CommonFields {
```

Comment:
it makes sense to put this interface in `types` folder

Reply:
fixed